docs/sources/next/extensions/explore.md

```diff
@@ -255,14 +219,10 @@ Use the table to explore the many extensions. Questions? Feel free to join the d
         <h4>xk6-output-timestream</h4>
         <p>Export results to AWS Timestream</p>
     </a>
-    <a href="https://github.com/wosp-io/xk6-playwright" target="_blank" class="nav-cards__item nav-cards__item--guide">
+    <a href="https://github.com/nicholasvuono/xk6-playwright" target="_blank" class="nav-cards__item nav-cards__item--guide">
         <h4>xk6-playwright</h4>
         <p>Browser automation and end-to-end web testing using Playwright</p>
     </a>
-    <a href="https://github.com/mcosta74/xk6-plist" target="_blank" class="nav-cards__item nav-cards__item--guide">
```

Comment:
The extension does not meet the registry requirements
https://github.com/mcosta74/xk6-plist/issues/2